.github/workflows/pull_request.yml

```diff
@@ -109,3 +109,95 @@ jobs:
       - uses: actions/checkout@v4
       - name: Check Spelling
         uses: crate-ci/typos@master
+
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Configure sccache
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Build xline image
+        run: |
+          docker run -q --rm -v $(pwd):/xline \
+            -e SCCACHE_GHA_ENABLED=on \
+            -e ACTIONS_CACHE_URL=${ACTIONS_CACHE_URL} \
+            -e ACTIONS_RUNTIME_TOKEN=${ACTIONS_RUNTIME_TOKEN} \
+            ghcr.io/xline-kv/build-env:latest \
+            cargo build --release --bin xline --bin benchmark --bin validation_lock_client
+
+          sudo apt-get install -y --force-yes expect
+          cd scripts
+          cp ../target/release/{xline,benchmark,validation_lock_client} .
```

Comment:
It seems the `validation_lock_client` does not need to be built.

Reply:
As the https://github.com/xline-kv/Xline/pull/696#discussion_r1527402223 say, just leave this to https://github.com/xline-kv/Xline/issues/700 ,  WDYT?